extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
Ah, i see. There are three cases:

1. No string
2. String specified in a list
3. Non-restricted string

for 1. we could add an additional argument to the `<event>` tag `payload=false` or something like this. No 2. then uses the `<event><options>...` syntax, and 3. uses <event/>

If i specify `<state />` it's also non restricted, right?